contracts/signal_oracle.py

```
@@ -0,0 +1,228 @@
+{{
+    "score": int,  // 0-100
+    "approved": bool,  // true if score >= {standard.approval_threshold}
+    "feedback": str,  // brief actionable feedback
+    "issues": [str]  // list of specific issues found
+}}
+It is mandatory that you respond only using the JSON format above, nothing else."""
+            
+            result = gl.nondet.exec_prompt(prompt, response_format="json")
+            return json.dumps(result, sort_keys=True)
+
+        eval_result = json.loads(gl.eq_principle.strict_eq(evaluate_signal))
```

Ruff (0.15.7)

[error] `gl` may be undefined, or defined from star imports

(F405)